v22.1/take-full-and-incremental-backups.md

```diff
@@ -92,45 +165,74 @@ Periodically run the [`BACKUP`][backup] command to take a full backup of your cl
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> BACKUP INTO '{destination}';
+> BACKUP INTO '{collectionURI}';
 ~~~
 
-Then, create nightly incremental backups based off of the full backups you've already created. To append an incremental backup to the most recent full backup created in the given destination, use `LATEST`:
+Then, create nightly incremental backups based off of the full backups you've already created. To append an incremental backup to the most recent full backup created at the [collection's](#backup-collections) URI, use the `LATEST` syntax:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> BACKUP INTO LATEST IN '{destination}';
+> BACKUP INTO LATEST IN '{collectionURI}';
 ~~~
 
-For an example on how to specify the destination of an incremental backup, see [Incremental backups with explicitly specified destinations](#incremental-backups-with-explicitly-specified-destinations).
+<span class="version-tag">New in v22.1:</span> This will add the incremental backup to the default `/incrementals` directory at the root of the backup collection's directory. With incremental backups in the `/incrementals` directory, you can apply different lifecycle/retention policies from cloud storage providers to the `/incrementals` directory as needed.
+
+{{site.data.alerts.callout_info}}
+In v21.2 and earlier, incremental backups were stored in the same directory as their full backup (i.e., `collectionURI/subdirectory`). If an incremental backup points to a subdirectory with incremental backups created in v21.2 and earlier, v22.1 will write the incremental backup to the v21.2 default location. To back up using the prior behavior, use the `incremental_location` option, as shown in this [example](#backup-earlier-behavior).
```

Comment:
nit: in the second sentence, add "command" , i.e. change to " If an incremental backup command points..."

per discussion in comments, perhaps clarify the last sentence to: "If you want to continue using the prior behavior on a backup that does not already contain incremental backups in the full backup subdirectory, use ..."